### DISCUSSION TITLE
21.07.30.금 회의록

### DISCUSSION THREAD

**Discussion by damilog** [Team-IT-DA/Frontend#15] [회의록]
- [x] ITDA 서비스 메인 color 정하기 (스타일 적용시 자주 쓰일)
- [x] font 정하기 
- [x] PR, Issue 템플릿 만들기 
- [ ] 검색어 자동 완성 구현 여부 정하기 - 백엔드와 논의 예정

**Comment by damilog** [comment]
다음 미팅:월요일 10시 반 미팅 

업무 분담 
### 제니 
- card, side drawer 부분 스타일 컨센션 적용 
- 시간 남으면 login page view 
### 디코 
- header 스타일 수정
- 브랜드 스토리 페이지 추가
### 데이지
- 검색창 cancel 버튼 눌렀을 때 리셋
- 카테고리 창 스타일 수정(위치, 색상 등)

**Reply by ha3158987** [reply]
이번 업무분담용으로 마일스톤 하나 만들어놨습니다~!
